src/parser/tests.rs

```diff
@@ -122,6 +134,15 @@ map<i32, i32> count;
     r#"wasm:bytecode:call:alt  / i == """" / { }"#,
     // bad statement
     "wasm:bytecode:call:alt / i == 1 / { i; }",
+    // bad incrementor
+    r#"
+    wasm:bytecode:br:before {
+        i = 0;
+        if(i++ == 0){
```

Comment:
I don't see why this is a bad incrementor (at least this is valid in C)

Reply:
It's not the wanted behavior for this DSL. Likely we want the "++" operator to not be accepted in expressions but as its own statement